components/mpas-ocean/src/Registry.xml

```diff
@@ -702,6 +702,39 @@
 					description="The length scale of exponential decay of river runoff. Fluxes are multiplied by $e^{z/\gamma}$, where this coefficient is $\gamma$."
 					possible_values="Any positive real number."
 		/>
+		<nml_option name="config_subglacial_runoff_mode" type="character" default_value="off"
+					description="Selects the mode in which subglacial runoff fluxes are implemented."
+					possible_values="'off', 'data'"
+		/>
+		<nml_option name="config_flux_attenuation_coefficient_subglacial_runoff" type="real" default_value="0.001" units="m"
+					description="The length scale of exponential decay of subglacial runoff. Fluxes are multiplied by $e^{z/\gamma}$, where this coefficient is $\gamma$."
+					possible_values="Any positive real number."
+		/>
+		<nml_option name="config_sgr_flux_vertical_location" type="character" default_value="bottom"
+					description="Selects the vertical location where subglacial runoff is fluxed."
+					possible_values="'top','uniform', 'bottom'"
+		/>
+		<nml_option name="config_use_sgr_opt_kpp" type="integer" default_value="1" units="non-dimensional"
+					description="option to use or not subglacial runoff (sgr) in kpp calculation"
+					possible_values="0 - don't use, 1 - use"
+		/>
+		<nml_option name="config_use_sgr_opt_temp" type="character" default_value="sgr"
```

Comment:
I like to keep the `config_use_*` options logical. So in this case it could be `config_use_sgr_temp_prescribed` or something like that.

Reply:
Changed as suggested.